CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## v6.0.0-beta.0 (Not yet published)
+## v6.1.0 (November 8, 2021)
```

Comment:
This should be 6.0.0; version needs to match the latest version here: https://www.npmjs.com/package/@pxblue/react-native-themes

Reply:
Also please change the date for tomorrow; should be able to get this merged in.

Reply:
Done.